csm/core/agent/csm_agent.py

```diff
@@ -135,6 +136,18 @@ def _configure_s3_services():
         CsmRestApi._app[const.S3_BUCKET_SERVICE] = BucketService(s3_plugin_obj)
         CsmRestApi._app[const.S3_CAPACITY_SERVICE] = S3CapacityService(s3_plugin_obj)
 
+    @staticmethod
+    def get_topology_config():
+        host = Conf.get(const.DATABASE_INDEX, const.CONSUL_CONFIG_HOST)
+        port = Conf.get(const.DATABASE_INDEX, const.DB_CONSUL_CONFIG_PORT)
+        index = Conf.get(const.CSM_GLOBAL_INDEX, const.TOPOLOGY_INDEX)
+        backend = Conf.get(const.CSM_GLOBAL_INDEX, const.TOPOLOGY_BACKEND)
+        topology_config = {
+            const.BACKEND : Conf.get(const.CSM_GLOBAL_INDEX, const.TOPOLOGY_OBJECT_STORE),
```

Comment:
const.BACKEND --> Change it to same as i mentioned above

Reply:
Corrected